google_health/delphi_google_health/export.py

```diff
@@ -27,6 +30,8 @@ def export_csv(
         name of the sensor; only used for naming the output file
     smooth: bool
         should the signal in "val" be smoothed?
+    start_date: str
```

Comment:
Hmm, in both `pull_api.py` and `export.py`, the vars related to date keeps the same string format of `YYYY-MM-DD`

Reply:
That's fine.  I will probably start an issue/conversation about migrating everything to dates rather than date strings, but it probably is too much to make localized changes here.